hdfs_namenode/tests/conftest.py

```diff
@@ -68,6 +76,12 @@ def raise_for_status(self):
             body = f.read()
             return MockResponse(body, 200)
 
+    elif args[0] == NAME_SYSTEM_METADATA_URL:
```

Comment:
Readability suggestion: since `args[0]` always seems to refer to an URL, should we update the signature to be…?

```python
def requests_get_mock(url, *args, **kwargs): ...
```